### PR TITLE
Release 0.1.0.

### PR DESCRIPTION
**Pull request by iphydf**
This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/316)

### CODE REVIEW

**Comment by zetok** [comment]
:lgtm_strong:

---

Review status: 0 of 4 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/316#-:-KYjQ8B_IUibdhdWcnLx:b-jbsf8z)*

**Comment by robinlinden** [comment]
:lgtm_strong:

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/316#-:-KYjiWFCzFQbCgDjGBTr:b-jbsf8z)*

**Comment by GrayHatter** [comment]
:lgtm_strong:

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/316#-:-KYkW7StvXlc59XcdJGM:b-jbsf8z)*

**Comment by JFreegman** [comment]
:lgtm_strong:

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/316#-:-KYpI_bSiie7h2dMUgMG:b-jbsf8z)*

**Comment by sudden6** [comment]
:lgtm_strong:

---

Review status: 0 of 4 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/316#-:-KYtsOdLefKOt7vgraZw:b-jbsf8z)*

File: configure.ac [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.65])
-AC_INIT([tox], [0.0.5])
+AC_INIT([tox], [0.1.0])
 AC_CONFIG_AUX_DIR(configure_aux)
```
Comment (jin-eld) [inline configure.ac]: Just to keep in mind.. if it's the "first" release you can probably leave it at 0:0:0, but make sure not to forget the LIBTOXCORE_LT_VERSION setting in configure.ac in future releases, it defines the library compatibility (this is passed to libtool as parameter) and has influence on the name of the library, i.e. .so.x.y.z,  I think cmake also has some setting for this.

**Comment by nurupo** [comment]
Given https://github.com/TokTok/c-toxcore/pull/325 makes it in, it should be fine. There are also some minor things I commented on in https://github.com/TokTok/c-toxcore/pull/310 https://github.com/TokTok/c-toxcore/pull/315.

:lgtm: 

---

Reviewed 3 of 4 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/316#-:-KYwZAkA-3CIjbvQCrqj:bxxm497)*

**Comment by nurupo** [comment]
:lgtm_strong:

---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/316#-:-KYwkLF5EeKELzZUb-kc:b-jbsf8z)*

**Comment by GrayHatter** [comment]
🎉

**Pull request merged**